File: app/src/main/java/com/example/ibrahim/servisinfo_ib150071/data/UpitiResultVM.java
package com.example.ibrahim.servisinfo_ib150071.data;

import java.io.Serializable;
import java.util.List;

/**
 * Created by Ibrahim on 28.7.2018..
 */

public class UpitiResultVM implements Serializable{

    public static class Row implements Serializable {
        public int UpitID;
        public String OpisKvara;
        public String MarkaUredjaja; // atribut marka je i za model i marku
        public String Naslov;
        public byte[] Slika;
        public String KlijentID;
    }
    public List<UpitiResultVM.Row> rows;

}

File: RKS_IB150071_WebServisi/RKS_IB150071_WebServisi/Model2/UpitiResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RKS_IB150071_WebServisi.Model2
{
    public class UpitiResultVM
    {
        public class Row
        {
            public int UpitID { get; set; }
            public string MarkaUredjaja { get; set; }
            public string OpisKvara { get; set; }
            public int KlijentID { get; set; }
            public string Naslov { get; set; }

        }
        public List<Row> rows;


    }
}